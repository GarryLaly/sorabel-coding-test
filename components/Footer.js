import React from "react";

const Footer = () => (
  <footer className="text-muted">
    <div className="container">
      <p className="float-right">
        <a href="#top">Back to top</a>
      </p>
      <p>Sorabel - this is project for coding skill test by Garry Priambudi</p>
      <p>
        My Github?{" "}
        <a href="https://github.com/GarryLaly">https://github.com/GarryLaly</a>.
      </p>
    </div>
  </footer>
);

Footer.propTypes = {};

export default Footer;
