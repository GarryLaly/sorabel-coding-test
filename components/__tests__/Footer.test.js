import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";

import Footer from "../Footer";

describe("Footer", () => {
  const setup = propOverrides => {
    const props = {
      ...propOverrides
    };

    const wrapper = shallow(<Footer {...props} />);

    return {
      props,
      wrapper
    };
  };

  it("should renders default", () => {
    const { wrapper } = setup();

    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
