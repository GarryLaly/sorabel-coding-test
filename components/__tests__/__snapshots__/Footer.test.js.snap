// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should renders default 1`] = `
<footer
  className="text-muted"
>
  <div
    className="container"
  >
    <p
      className="float-right"
    >
      <a
        href="#top"
      >
        Back to top
      </a>
    </p>
    <p>
      Sorabel - this is project for coding skill test by Garry Priambudi
    </p>
    <p>
      My Github?
       
      <a
        href="https://github.com/GarryLaly"
      >
        https://github.com/GarryLaly
      </a>
      .
    </p>
  </div>
</footer>
`;
